### PR TITLE
58 refactor

### PR DESCRIPTION
**Pull request by fuzzyers**
Refactored code from 58 arcade css to make it slightly more readable aswell as added in code from 64 arcade importing.

### CODE REVIEW

**Review by OP-ReubenK — APPROVED** [review]
awesome thanks for doing that

File: arcadebackend/index.js
Comment (SamanthaCra) [inline arcadebackend/index.js:1]: Is there any sort of reset for the next user to come along and not still be marked as over18?

File: arcadefrontend/src/components/Game.js
Comment (SamanthaCra) [inline arcadefrontend/src/components/Game.js:1]: Is this file still needed?

File: arcadefrontend/src/components/Carousel.js
Comment (SamanthaCra) [inline arcadefrontend/src/components/Carousel.js:1]: Is this file still needed?

File: arcadefrontend/src/styles/Carousel.css
Comment (SamanthaCra) [inline arcadefrontend/src/styles/Carousel.css:1]: Is this file still needed?

**Review by SamanthaCra — COMMENTED** [review]
A few files to clean up and a question on how the age restriction works without RFID working but otherwise looking good.

**Review by SamanthaCra — APPROVED** [review]
